tests/path_import/test_path_import.sh

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
+
+# $2 is sys.path from caller
+export PYTHONPATH="test_module_dir:$2"
+SCRIPT_DIR="$(dirname $0)"
+
+if [ "$1" = "fail" ]
```

Comment:
Guess you can simplify this with `python $SCRIPT_DIR/path_import.py "$1"`

Reply:
Definitely. Will do!